### DISCUSSION TITLE
Include order issue

### DISCUSSION THREAD

**Discussion by rvdbreemen** [bxparks/AceTime#76] [General]
Well, interestingly enough I struggled with the order of the includes. 

So I am using TimeLib.h next to AceTime.h. I do this to minimize code change... Historically I used ezTime and before that TimeLib. Anyway, there is lots of compile errors if I include AceTime later dan TimeLib. 

So the order that worked for me is:
#include <AceTime.h>
#include <TimeLib.h>

There must be a namespace conflict or something, because if I do it the otherway around I won't even compile.

What could be the reason for that?

The reason why I included TimeLib.h again,is that AceTime does not support the normal now(), hour(), minute(), etc. API.

Would it be thinkable to support a "default Timezone" that you setup on the object, so that the "TimeLib" API would also work?

**Comment by bxparks** [comment]
Could you please avoid filing duplicate issues, and bombarding me with so many at the same time?
Dealing with each request consumes my limited time and energy. Dealing with so many at the same time is incredibly demotivating.

**Reply by bxparks** [reply]
Duplicate of https://github.com/bxparks/AceTime/issues/77
